tests/test_wave_03.py

```diff
@@ -55,12 +55,14 @@ def test_friends_unique_movies_not_duplicated():
     # Assert
     assert len(friends_unique_movies) == 3
 
-    raise Exception("Test needs to be completed.")
+    # raise Exception("Test needs to be completed.")
     # *************************************************************************************************
     # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
+    assert friends_unique_movies == [FANTASY_4, HORROR_1, INTRIGUE_3]
+    assert amandas_data["watched"] not in friends_unique_movies
```

Comment:
So, this assert statement here is checking if the **entire** list of watched movies is **a single element** inside of `friends_unique_movies`. How could we change this to check if the individual movies are inside of `friends_unique_watched`?